sql/moz-fx-data-shared-prod/ltv_state_values_derived/dataset_metadata.yaml

```
@@ -0,0 +1,14 @@
+friendly_name: LTV State Values Derived
+description: |
+  Dataset where the data science team has write access to write LTV state values from their models
+dataset_base_acl: derived_restricted
```

The configuration you've written would create a dataset that DE doesn't have write access to: only DS would. Assuming we trust DE not to accidentally delete the data, and since the data is presumably not sensitive (it was in the `analysis` dataset which is `workgroup:mozilla-confidential`) the standard archetype + DE write access would be implemented by:
```suggestion
dataset_base_acl: derived
```